source/extensions/extensions_metadata.yaml

```diff
@@ -735,3 +740,8 @@ envoy.rbac.matchers.upstream_ip_port:
   - envoy.rbac.matchers
   security_posture: unknown
   status: alpha
+envoy.http.stateful_session.cookie:
+  categories:
+  - envoy.http.stateful_session
```

Comment:
I'm not sure about this category name. Should we ask around?

Reply:
I have no preference for this, do you have any suggestions?